vip-plugins/vip-plugins.php

```diff
@@ -222,20 +215,13 @@ function wpcom_vip_site_option_active_sitewide_plugins( $value, $option ) {
 function wpcom_vip_pre_update_option_active_plugins( $value, $old_value, $option ) {
```

Comment:
The `wpcom_vip_pre_update_option_active_plugins ` and `wpcom_vip_pre_update_site_option_active_sitewide_plugins` are identical now. Except for the number of args it accepts. Perhaps we could merge those two into a single function and have the `$network_id ` param being set to 0 in case that param is not provided by `pre_update_option_active_plugins` filter? It would make the code cleaner.

Reply:
I think we need them to be different as per above.